scheduler.go

```diff
@@ -38,19 +61,10 @@ func selectUnlaunchedInstances(instances []ecs_state.ContainerInstance, newTask
 }
 
 // StartTask on AWS ECS Cluster
-// cluster the name of cluster
 // taskArn the ARN of task or the name of task
-func StartTask(cluster string, taskDefinitions []string, awsRegion string) error {
-	// List cluster info
-	sess, _ := session.NewSession(&aws.Config{Region: aws.String(awsRegion)})
-	client := ecs.New(sess)
-
-	state := ecs_state.Initialize(cluster, client)
-	state.RefreshClusterState()
-	state.RefreshContainerInstanceState()
-	state.RefreshTaskState()
-
-	clusterInstance := state.FindClusterByName(cluster)
+func StartTask(taskDefinitions []string) error {
+	// Cluster comes from global variable which is determined on the start of runtime of scheduler.
```

Comment:
Hmm i think let's do it next. I think we should refresh more often than that, perhaps just 1 minute.